monkey/infection_monkey/ransomware/ransomware_readme.txt

```diff
@@ -1,2 +1,49 @@
-This is a placeholder README for the Infection Monkey Ransomware Simulation.
-Don't panic :)
+ ██████████                        ██  █████       ███████████                        ███
+░░███░░░░███                      ███ ░░███       ░░███░░░░░███                      ░░░
+ ░███   ░░███  ██████  ████████  ░░░  ███████      ░███    ░███  ██████   ████████   ████   ██████
+ ░███    ░███ ███░░███░░███░░███     ░░░███░       ░██████████  ░░░░░███ ░░███░░███ ░░███  ███░░███
+ ░███    ░███░███ ░███ ░███ ░███       ░███        ░███░░░░░░    ███████  ░███ ░███  ░███ ░███ ░░░
+ ░███    ███ ░███ ░███ ░███ ░███       ░███ ███    ░███         ███░░███  ░███ ░███  ░███ ░███  ███
+ ██████████  ░░██████  ████ █████      ░░█████     █████       ░░████████ ████ █████ █████░░██████
+░░░░░░░░░░    ░░░░░░  ░░░░ ░░░░░        ░░░░░     ░░░░░         ░░░░░░░░ ░░░░ ░░░░░ ░░░░░  ░░░░░░
```

Comment:
Maybe "Infection Monkey was here" would be better? In my mind, infection monkey spreading across the network is a valid reason to panic :D

Reply:
I'm having trouble getting "Infection Monkey was here" in a readable font. Anything easily readable comes out too large.

Since this PR has approvals, I'm going to merge it without this change. I'll have our marketing team look at this (and our documentation) before we release. They'll probably come up with something better.